### PR TITLE
Conversion lat and lon graus to decimal

### PR DESCRIPTION
**Pull request by MCavalcante16**
## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [x] I have read the **CONTRIBUTING** document.
-   [x] My code follows the code style of this project (see `.code-style.md`).
-   [x] All new and existing tests passed (see `.testing.md`).
-   [x] I have added tests to cover my changes.
-   [x] My change requires a change to the documentation.
-   [x] I have updated the documentation accordingly (see `.documentation.md`).

## Description

-   What is the new behavior (if this is a feature change)?
    Converts latitude and longitude format from
    decimal degrees to decimal format.

### CODE REVIEW

File: pymove/utils/conversions.py [hunk 1]
```diff
@@ -288,6 +288,55 @@ def geometry_points_to_lat_and_lon(
...
+    copy_lon = move_data[longitude].copy()
+    for i in move_data.index:
+        if (move_data[latitude][i][-1:] == 'N'):
+            move_data[latitude][i] = float(copy_lat[i][:-1])
+        else:
+            move_data[latitude][i] = float(copy_lat[i][:-1]) * -1
+        if (move_data[longitude][i][-1:] == 'E'):
+            move_data[longitude][i] = float(copy_lat[i][:-1])
+        else:
+            move_data[longitude][i] = float(copy_lon[i][:-1]) * -1
```
Comment (flych3r) [inline pymove/utils/conversions.py:334]: Tentar vetorizar o código.

**Review by flych3r — CHANGES_REQUESTED** [review]

File: pymove/utils/distances.py [hunk 2]
```diff
@@ -58,6 +59,34 @@ def haversine(lat1, lon1, lat2, lon2, to_radians=True, earth_radius=6371):
         raise e
 
 
+def distance_in_meters(lat1, lon1, lat2, lon2):
```
Comment (flych3r) [inline pymove/utils/distances.py:62]: Melhor renomear para `euclidean_distance`

File: pymove/tests/test_utils_distances.py [hunk 3]
```diff
@@ -72,6 +72,13 @@ def test_haversine():
 
     assert_almost_equal(dist, expected)
 
+def text_distance_in_meters():
```
Comment (flych3r) [inline pymove/tests/test_utils_distances.py:75]: Está escrito `text` ao invés de `test`

**Review by flych3r — CHANGES_REQUESTED** [review]

File: pymove/utils/distances.py (hunk 2 above)
Comment (MCavalcante16) [reply pymove/utils/distances.py:63]: euclidean_distance_in_meters seria melhor, não? a distancia euclidiana pura não é dessa forma.

File: pymove/tests/test_utils_distances.py [hunk 4]
```diff
@@ -72,6 +72,13 @@ def test_haversine():
 
     assert_almost_equal(dist, expected)
 
+def test_distance_in_meters():
```
Comment (flych3r) [inline pymove/tests/test_utils_distances.py:75]: Lembra que o teste deve ter o mesmo nome da função que está tratando, só com o prefixo `test_`

**Review by flych3r — CHANGES_REQUESTED** [review]

**Review by flych3r — APPROVED** [review]

**Pull request merged**